examples/powertools-examples-sqs/src/main/java/org/demo/sqs/SqsPoller.java

```diff
@@ -0,0 +1,60 @@
+package org.demo.sqs;
+
+import java.util.Random;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.RequestHandler;
+import com.amazonaws.services.lambda.runtime.events.SQSEvent;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import software.amazon.awssdk.http.urlconnection.UrlConnectionHttpClient;
+import software.amazon.awssdk.services.sqs.SqsClient;
+import software.amazon.lambda.powertools.logging.Logging;
+import software.amazon.lambda.powertools.sqs.SqsBatch;
+import software.amazon.lambda.powertools.sqs.SqsMessageHandler;
+import software.amazon.lambda.powertools.sqs.SqsUtils;
+
+import static com.amazonaws.services.lambda.runtime.events.SQSEvent.SQSMessage;
+
+/**
+ * Handler for requests to Lambda function.
+ */
+public class SqsPoller implements RequestHandler<SQSEvent, String> {
+
+    Logger log = LogManager.getLogger(SqsPoller.class);
+    Random random = new Random(100);
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

To make your code secure, use `new SecureRandom()` to instantiate a `java.security.SecureRandom` object. Do not use `getInstance()`, `getInstanceStrong()` and `java.util.Random`.